lxd/devlxd.go

```diff
@@ -438,7 +439,8 @@ func findContainerForPid(pid int32, d *Daemon) (container, error) {
 			parts := strings.Split(string(cmdline), " ")
 			name := strings.TrimSuffix(parts[len(parts)-1], "\x00")
 
-			return containerLoadByName(d.State(), name)
+			// FIXME[projects]: how to figure out the project name here?
+			return containerLoadByProjectAndName(d.State(), "default", name)
```

Comment:
How crucial is this for a correctly working `devlxd`? :)

Reply:
Probably pretty crucial actually, not fixing this would actually be a security issue as it'd allow for a container in a project that has `security.devlxd=false` to access the /dev/lxd API of a similarly named container in the default profile (both exposing an attack surface to LXD to an untrusted container and allowing an information leak between two containers).

Reply:
Oh yeah, I forgot to highlight this fact in the PR description. Good catch. Any idea how we could fix this?

Reply:
Doesn't the project appear in the `[lxc monitor] ...` process string? (as project_name)

Reply:
@stgraber, iff the rootfs path does have the project appended but yeah, it should. We can also introduce a new config setting (I thought about this for a while): `lxc.monitor.proctitle` or something that would let users set a name for the process.